app/src/main/java/ch/admin/bag/dp3t/util/DateUtils.java

```diff
@@ -20,9 +23,10 @@
 
 	private static final DateFormat DATE_TIME_FORMAT = SimpleDateFormat.getDateTimeInstance();
 	private static final DateFormat DATE_FORMAT = SimpleDateFormat.getDateInstance();
+	@SuppressLint("SimpleDateFormat")
+	private static final DateFormat DATE_LAST_UPDATED_FORMAT = new SimpleDateFormat("yyyy-MM-dd");
```

Comment:
I'd specifically add the `Locale.ENGLISH` to avoid issues when parsing the date on systems with non-ascii locales. Also the `@SuppressLint("SimpleDateFormat")` can then be removed.

Reply:
Done, and also moved inline to satisfy SonarCloud